storage/sql/migrate.go

```diff
@@ -174,6 +174,9 @@ var migrations = []migration{
 			alter table refresh_token
 				add column token text not null default '';`,
 			`
+			alter table refresh_token
```

Comment:
This is not right, we should add a new migration entry, at the end of the list, containing just this statement.

Reply:
Oh, yeah. Migrations... Fixed.